src/v2/Utils/user.ts

```diff
@@ -35,3 +36,9 @@ export function userIsAdmin(user?: User): boolean {
   const isAdmin = Boolean(user && user.type === "Admin" ? true : false)
   return isAdmin
 }
+
+export function userHasAccessToPartner(user: User, partnerId: string): boolean {
+  const token = get(user, u => u.accessToken)
+  const decodedToken = jwt_decode(token)
+  return decodedToken.partner_ids.includes(partnerId)
```

Comment:
I like this approach! One caveat that we might want to document somewhere for our business stakeholders is that this list of `partner_ids` on the token is subject to drift a bit from the active permissions in the database.

For example:

- A user logged into Force who gains access to a partner account would have to reauthenticate for the token to reflect the new partner ID, gaining access to VR preview mode.
- A user logged into Force who loses access to a partner account would have to reauthenticate before they lose access to the VR preview mode.